python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -373,10 +373,17 @@ async def run_cwfs(self):
             self.log.info(
                 f"Running cwfs in " f"{self.intra_visit_id}/{self.extra_visit_id}."
             )
+            self.log.info(
```

Comment:
Consider using debug level for these messages.

Reply:
I changed a bunch of them throughout and nearby... but actually left this specific instance.